app/src/main/java/org/oppia/app/profile/AddProfileActivityPresenter.kt

```diff
@@ -112,9 +146,9 @@ class AddProfileActivityPresenter @Inject constructor(
       }
     }
 
-    binding.inputName.setInput(profileViewModel.inputName.get().toString())
-    binding.inputPin.setInput(profileViewModel.inputPin.get().toString())
-    binding.inputConfirmPin.setInput(profileViewModel.inputConfirmPin.get().toString())
+    binding.addProfileActivityUserNameProfileInputView.setInput(profileViewModel.inputName.get().toString())
+    binding.addProfileActivityInputPinProfileInputView.setInput(profileViewModel.inputPin.get().toString())
+    binding.addProfileActivityInputConfirmPinProfileInputView.setInput(profileViewModel.inputConfirmPin.get().toString())
```

Comment:
Can we rename these two input pin views to `addProfileActivityPinProfileInputView` and `addProfileActivityConfirmPinProfileInputView`. Just a suggestion.

Reply:
Done.